<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="home-laptop" failures="2" tests="11" name="Test" time="18.954" errors="0" timestamp="2023-05-31T13:22:57 IST">
  <testcase classname="com.notesapi.tests.UsersTests" name="createNewUser" time="8.025"/>
  <testcase classname="com.notesapi.tests.UsersTests" name="performLogin" time="0.799">
    <failure type="java.lang.AssertionError" message="expected [201] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at com.notesapi.tests.UsersTests.performLogin(UsersTests.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- performLogin -->
  <testcase classname="com.notesapi.tests.UsersTests" name="retrieveUserInfo" time="3.024"/>
  <testcase classname="com.notesapi.tests.UsersTests" name="editUserInfo" time="0.491">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at com.notesapi.tests.UsersTests.editUserInfo(UsersTests.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- editUserInfo -->
  <testcase classname="com.notesapi.tests.UsersTests" name="editUserPassword" time="0.925"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="createNewNote" time="0.86"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="retrieveNote" time="0.582"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="retrieveNoteById" time="0.839"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="updateExistingNote" time="0.698"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="updateCompleteStatusOfNote" time="0.745"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="deleteNoteById" time="0.728"/>
</testsuite> <!-- Test -->
